.github/ISSUE_TEMPLATE/1_bug_report.yaml

```
@@ -1,43 +1,18 @@
 name: Report a bug
 description: Any errors that are not UI related.
-labels: ["needs triage", "bug"]
+labels: ["needs triage"]
 body:
-  - type: markdown
-    attributes:
-      value: >
-        WAIT!
-
-        Before you go any further. Is this really a **🐛 bug**?
-
-        If it's a question about how Lightning works, have a look at our [Lightning documentation](https://lightning.ai/lightning-docs/)
-        or ask a question on our [Community Slack](https://pytorch-lightning.slack.com).
-
-        If it really is a **🐛 bug**, please fill out this form as completely as you can.
-
-  - type: checkboxes
-    id: checks
-    attributes:
-      label: First check
-      description: Please confirm and check all the following options.
-      options:
-        - label: I'm sure this is a bug.
-          required: true
-        - label: I've added a descriptive title to this bug.
-          required: true
-        - label: I've provided clear instructions on how to reproduce the bug.
-          required: true
-        - label: I've added a code sample.
-          required: true
-        - label: I've provided any other important info that is required.
-          required: true
-
   - type: textarea
     attributes:
       label: Bug description
-      description: A clear and concise description of the 🐛bug🐛.
+      description: A description of the 🐛bug🐛.
```

Maybe we can add a placeholder?
```suggestion
      description: A description of the 🐛bug🐛.
    placeholder: |
      A clear and concise description of what the bug is.
      
      Include steps to reproduce, the expected behavior, and the actual behavior.
      
       ```
      The error message you got, with the full traceback if available.
      ```      
```